diff --git a/test/MatrixTranspose_test/Makefile b/test/MatrixTranspose_test/Makefile
index 758f8d9..f7127d7 100644
--- a/test/MatrixTranspose_test/Makefile
+++ b/test/MatrixTranspose_test/Makefile
@@ -4,8 +4,14 @@ LIB_PATH  ?= $(ROOT_PATH)/build
 ROCM_PATH ?= /opt/rocm
 HIP_VDI ?= 0
 ITERATIONS ?= 100
-
-ROC_LIBS  = -Wl,--rpath,${LIB_PATH} $(LIB_PATH)/libroctracer64.so $(LIB_PATH)/libroctx64.so
+C_TEST=0
+ROC_LIBS  = -Wl,--rpath,${LIB_PATH} \
+    $(LIB_PATH)/libroctracer64.so \
+    $(LIB_PATH)/libroctx64.so \
+    $(LIB_PATH)/libkfdwrapper64.so \
+    $(ROCM_PATH)/hip/lib/libamdhip64.so \
+    $(ROCM_PATH)/lib/libamdhip64.so \
+    $(ROCM_PATH)/rocprofiler/lib/librocprofiler64.so
 
 HIP_PATH ?= $(wildcard /opt/rocm/hip)
 ifeq (,$(HIP_PATH))
@@ -17,7 +23,12 @@ TARGET=hcc
 
 EXECUTABLE=./MatrixTranspose
 OBJECTS = MatrixTranspose.o
-FLAGS =-g $(INC_PATH:%=-I%) -I$(ROCM_PATH)/hsa/include/hsa -I$(ROCM_PATH)/hsa/include -I$(ROCM_PATH)/hip/include -I$(ROCM_PATH)/include -DLOCAL_BUILD=1 -DHIP_VDI=${HIP_VDI} -DITERATIONS=$(ITERATIONS) -DAMD_INTERNAL_BUILD=1
+FLAGS =-g $(INC_PATH:%=-I%) \
+    -I$(ROCM_PATH)/roctracer/include/ \
+    -I$(ROCM_PATH)/hsa/include/hsa -I$(ROCM_PATH)/hsa/include \
+    -I$(ROCM_PATH)/hip/include \
+    -I$(ROCM_PATH)/include \
+    -DLOCAL_BUILD=1 -DHIP_VDI=${HIP_VDI} -DITERATIONS=$(ITERATIONS) -DAMD_INTERNAL_BUILD=1
 
 ifeq ($(C_TEST), 1)
 	COMP=${CC}
